octoprint_printhistory/parser_test/README.txt

```diff
@@ -0,0 +1 @@
+Do not use those files for printing, they are prepared for testing purposes.
```

Comment:
that's not a sufficient README by all means. Take some good plain boilerplate like this https://gist.github.com/PurpleBooth/109311bb0361f32d87a2 and rewrite yours